app.js

```diff
@@ -11,56 +11,74 @@ const totalEl = document.getElementById('total');
 const lossesEl = document.getElementById('losses');
 const winsEl = document.getElementById('wins');
 
-
 let correctGuesses = 0;
 let totalGuesses = 0;
+let incorrectGuesses = 0;
```

Comment:
here you can also derive the incorrect state from the other two states like this:  `let incorrectGuesses = totalGuesses - incorrectGuesses` but how you did it works as well!